storage/entities/transaction.js

```diff
@@ -342,6 +346,8 @@ class Transaction extends BaseEntity {
 			t.signatures = t.signatures ? t.signatures.join() : null;
 			t.amount = t.amount.toString();
 			t.fee = t.fee.toString();
+			// Set default create values
+			_.defaults(t, defaultCreateValues);
```

Comment:
```suggestion
			t.recipientId = t.recipientId || null;
```

Reply:
Implemented the change directly as it also involved changing other lines but same change as suggested. Resolving.